controllers/replication.storage/volumereplication_controller.go

```diff
@@ -363,13 +365,30 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 
 	instance.Status.LastCompletionTime = getCurrentTime()
+	var last_sync_time int64
+	var requeueForInfo bool
+	if instance.Spec.ReplicationState == replicationv1alpha1.Primary {
+		last_sync_time, err = r.getVolumeReplicationInfo(vr)
+		last_sync_time1 := time.Unix(last_sync_time, 0).UTC()
+		instance.Status.LastSyncTime = getTime(last_sync_time1)
+		requeueForInfo = true
+
+	}
 	err = r.updateReplicationStatus(instance, logger, getReplicationState(instance), msg)
 	if err != nil {
 		return ctrl.Result{}, err
 	}
 
 	logger.Info(msg)
 
+	if requeueForInfo {
```

Comment:
why add requeue? I guess you want a recurring call to regularly update the state?

Reply:
yes, requeue should do that right? IT will requeue after every interval.